explainer.md

```diff
@@ -134,7 +134,73 @@ The Realms API does __not__ introduce a new evaluation mechanism. The code evalu
 
 JavaScript modules are associated with a global object and set of built-ins. Realms contain their own separate module graph which runs in the context of that Realm, so that a full JavaScript development experience is available.
 
-## <a name='Clarifications'></a>Clarifications
+## Clarifications from the TAG Review
+
+This proposal has been submitted for a [TAG Review](https://github.com/w3ctag/design-reviews/issues/542) and the following subtopics represent some of the clarifications [we presented](https://github.com/w3ctag/design-reviews/issues/542#issuecomment-698615062).
+
+### So do Realms only have the ECMAScript APIs available?
```

Comment:
maybe moving this to a FAQs section at the end of the explainer, or something like that, is better, these are generic enough to not need to mention the tag review.

Reply:
Done